app/views/content/landing/career-progression/_collage.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-full-content">
+      <div class="inset">
+        <p class="col-720">
+Teaching is fulfilling career. Not only can you help the pupils you teach, you also get to learn new skills, build your confidence and become your best self. The benefits of being a teacher include:         </p>
+
+        <%= render Content::ChecklistCollageComponent.new(
+          checklist: [
+            "career progression ",
+            "personal development",
+            "teacher's pay"
+          ],
+          image_paths: [
+            "static/content/how-much-do-teachers-get-paid/collage2.jpg",
+          ]
+        ) do %>
+        <% end %>
+      </div>
+    </section>
```

Comment:
there is a lot of blank space at the end of this component (not just on this landing page, but on all of them) - might be one for @garethwilliamsdesign to have a look at in a separate ticket